### PR TITLE
Add golang-115 and make default

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

File: components/developer/golang-115/pkg5 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "dependencies": [
+        "developer/golang-113",
+        "SUNWcs",
+        "system/library"
+    ],
+    "fmris": [
+        "developer/golang-114"
+    ],
```
Comment (AndWac) [inline components/developer/golang-115/pkg5:10]: There is something wrong here!

Comment (Toasterson) [reply components/developer/golang-115/pkg5:10]: Jep. How does that file get generated?

Comment (AndWac) [reply components/developer/golang-115/pkg5:10]: It's generated by gmake update-metadata (part of gmake publish)

Comment (Toasterson) [reply components/developer/golang-115/pkg5:10]: Fixed

**Comment by AndWac** [comment]
I have a similar problem with this PR like with #6047:
╰─➤  diff sample-manifest.p5m sample-manifest.p5m.orig
203a204
> file path=usr/lib/golang/1.15/gocache/06/06b3989957976a6b2723a14cfc4c4377ceae76558a21e79ce6495ab8e11471e6-a
205,207c206,208
< file path=usr/lib/golang/1.15/gocache/50/5060c63f6807170c9e43452103b91450c19a45d79a653775dff443f0e064778a-a
< file path=usr/lib/golang/1.15/gocache/7b/7b5cddfd2ca850f83386ffc1a8acb26850fa738a42d4c9e07ceb7b9d893d6775-a
< file path=usr/lib/golang/1.15/gocache/9b/9be4f6d80223575e2466787ef253aa939ec752500e297de53ee29a164dcca301-a
---
> file path=usr/lib/golang/1.15/gocache/4f/4f5b1726b088bbd9407ae822674108fc2f87f56a5e0ec37ac80fc193ef2a1d03-d
> file path=usr/lib/golang/1.15/gocache/94/94bd909e13705797e06ef73ad8d1ebf33aef1c7cc1cdb3a014d6f03c24c681ab-a
> file path=usr/lib/golang/1.15/gocache/9d/9dc47dabe4f953189e8d061e37531d8bd75477dbf8930859d329c85e7727edcd-a
209c210
< file path=usr/lib/golang/1.15/gocache/ba/ba352063887c06c0f19852be28a8dad44e5ced816a13a17a954af783a8396729-a
---
> file path=usr/lib/golang/1.15/gocache/a7/a7793b786daf4e61a35e0208c14d5b35bc08c8ed52e5e0ea24edbeed98587a58-a
211d211
< file path=usr/lib/golang/1.15/gocache/f6/f664af7d89603d77665aa766a386f9c36c8d9974ecfec042bb7c18b3f9f1e51a-d

These different files prohibit a successful publishing of the package. This needs to be solved before being able to merge.

**Comment by Toasterson** [comment]
Oh yes thanks for the reminder. it's a know thing with the golang packages. In short everything below gocache folder needs to be taken out of the manifest as the filenames change on every build by design. Linux Packaging software do not look at every file individually and thus it only affects us. We did not deliver gocache as golang does not need it at all to run. I'll clean that up tomorrow.

**Comment by AndWac** [comment]
Also publishes now. Please squash the commits and I will merge the PR.

**Pull request merged**